### DISCUSSION TITLE
示例代码中的非注释部分是否需要翻译的疑问

### DISCUSSION THREAD

**Discussion by earthaYan** [reactjs/zh-hans.react.dev#1216] [Q&A] [answered]
review [PR](https://github.com/reactjs/zh-hans.react.dev/pull/1144)的时候，发现这个PR的示例代码非注释部分也翻译了，指南里面有提到 [关于代码的翻译](https://github.com/reactjs/zh-hans.react.dev/wiki/React-Translation-Guide)， 但是查看了[中文网站](https://zh-hans.react.dev/learn/thinking-in-react)一些已合并的[内容](https://zh-hans.react.dev/learn/describing-the-ui)，非注释部分代码并没有翻译，所以想了解下代码的内容是否需要翻译？
个人理解示例代码非注释部分应该是不需要翻译的，它属于源码，不包含难以理解的长难句，且出现的名词基本都是常见单词。这种由于拼接问题翻译成中文语序读起来好像也有点变扭

> console.log('✅ Connecting to "' + roomId + '" room at ' + serverUrl + '...');
      console.log('✅ 连接到 "' + roomId + '" 房间，在' + serverUrl + '...');

**Comment by awxiaoxian2020** [comment]
请调整语序进行翻译

**Reply by Huxpro** [reply]
@awxiaoxian2020 目前的规则要翻还是不翻译？我个人觉得是可以不翻其实？

**Reply by Yucohny** [reply]
目前是都翻译的。

**Reply by Huxpro** [reply]
@Yucohny 我看不少章节 UI 和代码里的字符串都没翻，但是比如 <https://zh-hans.react.dev/learn/referencing-values-with-refs> 又翻了，感觉需要统一一下标准？都翻本身没问题 but it's a really high standard

**Reply by Yucohny** [reply, marked as answer]
一方面似乎之前没有明确说明需要翻译，一方面 reviewer 可能也没注意，造成了有些章节翻译了有些章节没有翻译的情况。考虑此举确实能让示例代码更加通俗易懂一些，我还是倾向于翻译。我在翻译规范中补充一下。
